src/test/java/service/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,46 @@
+package service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import service.OperationHandler;
+import service.OperationStrategy;
+import strategy.BalanceOperation;
+import strategy.PurchaseOperation;
+import strategy.ReturnOperation;
+import strategy.SupplyOperation;
+
+class OperationStrategyImplTest {
+    private OperationStrategy operationStrategy;
+    private final OperationHandler balanceOperation = new BalanceOperation();
+    private final OperationHandler purchaseOperation = new PurchaseOperation();
+    private final OperationHandler returnOperation = new ReturnOperation();
+    private final OperationHandler supplyOperation = new SupplyOperation();
```

Comment:
initialise BeforeAll 
```suggestion
    private final OperationHandler balanceOperation = new BalanceOperation();
    private final OperationHandler purchaseOperation = new PurchaseOperation();
    private final OperationHandler returnOperation = new ReturnOperation();
    private final OperationHandler supplyOperation = new SupplyOperation();
```